server/db.json

```diff
@@ -23,7 +23,7 @@
     },
```

Comment:
did you really intend to update this reference data? seems unrelated to the portfolio count.